app/backend/core/message_truncater/images_helper.py

```diff
@@ -0,0 +1,64 @@
+import base64
+import math
+import re
+from io import BytesIO
+
+from PIL import Image
+
+"""
+This function checks whether the base code encoded image is a valid base64 string
+and returns the dimensions of the image.
+"""
+
+
+def get_image_dims(image_uri: str) -> tuple[int, int]:
```

Comment:
Let's make this a proper docstring (underneath the function signature, and with proper formatting).